### ISSUE TITLE
Session disappears immediately with large TTL

### ISSUE THREAD

**Issue by tleyden** [couchbase/sync_gateway#1049]
To reproduce:

Run sync gw with [this config](https://gist.github.com/tleyden/5f13c8d2645460b1fc96)

Add a user:

```
curl -X "POST" "http://localhost:4985/db/_user/" \
    -d "{\"name\":\"foo@bar.com\",\"password\":\"password\",\"admin_channels\":[\"public\"],\"admin_roles\":[\"TestRole\"]}"

```

Then add a session with a large TTL (3600000 or roughly 41 days) via:

```
curl -X "POST" "http://localhost:4985/db/_session" \
    -d "{\"name\":\"foo@bar.com\",\"password\":\"password\",\"ttl\":3600000}"
```

**Expected**: there should be a session document in couchbase server
**Actual**: either no session document is created, or it expires immediately

OTOH, with a smaller TTL (360000 seconds or roughly 4 days), the session creation works as expected:

```
curl -X "POST" "http://localhost:4985/db/_session" \
    -d "{\"name\":\"foo@bar.com\",\"password\":\"password\",\"ttl\": 360000}"
```

Versions used:
- Sync Gateway 1.1.1
- Couchbase Server 3.0.3

**Comment by ajres** [comment]
Duplicate of #974

**Comment by tleyden** [comment]
Workaround fix: if the session is greater than 30 days, cap it down to 30 days and log a warning.  Deeper fix:  #974
